src/dmd/declaration.d

```diff
@@ -1097,11 +1097,18 @@ extern (C++) class VarDeclaration : Declaration
 
     VarDeclarations* maybes;        // STC.maybescope variables that are assigned to this STC.maybescope variable
 
+    private bool isAnonymous_;
```

Comment:
What does this do? Why does it have a trailing _ ?

Reply:
It has a trailing `_` to avoid a conflict with the method `isAnonymous`. It indicates that variable is anonymous that therefore will not be added to the symbol table.

Reply:
The [convention](https://dlang.org/dstyle#naming_keywords) is to use `name_` (instead of just `name`) when `name` is a keyword. If the variable is aggregate-level (module, class, etc.) private it [can be prefixed with an underscore](https://dlang.org/dstyle#naming_general).
Since the `_name` style is used in other languages (in addition to being allowed by the style guide) I suggest:
```suggestion
    private bool _isAnonymous;
```

Reply:
Fine, but let the record show I do this change under protest 😃.

Reply:
Fixed.